src/infrastructure/figma/figma-service.ts

```diff
@@ -413,10 +424,16 @@ export class FigmaService {
 					credentials.accessToken,
 				);
 
+				const fileMetaResponse = await figmaClient.getFileMeta(
```

Comment:
**suggestion:** Please, consider sending requests concurrently. See the comment above for more detail.